.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: cimg/python:3.9.2
+      - image: circleci/mongo:latest-ram
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - dev-cache-{{ checksum "tox.ini" }}
+      - run:
+          name: Install dev dependencies
+          command: pip3 install tox tox-pyenv
+      - run: pyenv install --skip-existing 3.5-dev
+      - run: pyenv install --skip-existing 3.6-dev
+      - run: pyenv install --skip-existing 3.7-dev
+      - run: pyenv install --skip-existing 3.8-dev
+      - run: pyenv install --skip-existing 3.9-dev
```

Comment:
Theoretically, this would be faster if these were all individual jobs, because it could run them all in parallel. We at least do that in another OSS Python Library project: https://github.com/ustudio/ustack-tornado-shutdown/blob/master/.circleci/config.yml

Probably not worth fixing at the moment, but I thought it was worth mentioning.